pyiron_base/jobs/job/extension/server/generic.py

```diff
@@ -441,6 +447,20 @@ def view_queues():
         else:
             return None
 
+    @property
+    def executor(self):
+        return self._executor
+
+    @executor.setter
+    def executor(self, exe):
+        if isinstance(exe, Executor):
+            self._executor = exe
+            self.run_mode.executor = True
+        else:
```

Comment:
```suggestion
        elif exe is None:
            self._executor = None
            self.run_mode.executor = False
        else:
```

Let's add a way to turn the executor back off if the user changes their mind.